.template/addons/crud/app/assets/stylesheets/layouts/index.scss

```
@@ -0,0 +1 @@
+@import "./application";
```

We can do like this, right? 😛 
```suggestion
@import "application";
```